README.md

```diff
@@ -144,7 +144,7 @@ Both of these are consequences of a desire to ensure that same-origin sites do n
 
 You can see a more full analysis of what results this algorithm produces in our [scenarios document](./scenarios.md).
 
-As far as the web-developer–observable consequences of using origins for agent cluster keys, this will automatically cause `SharedArrayBuffer`s to no longer be shareable cross-origin, because of the agent cluster check in [StructuredDeserialize](https://html.spec.whatwg.org/multipage/structured-data.html#structureddeserialize). We'd also need to add a check to the [`document.domain` setter](https://html.spec.whatwg.org/multipage/origin.html#dom-document-domain) to make it throw. (Ideally we'd find some way of restructuring the existing checks in `document.domain` so that they are also tied to agent cluster boundaries.)
+As far as the web-developer–observable consequences of using origins for agent cluster keys, this will automatically cause `SharedArrayBuffer`s to no longer be shareable cross-origin, because of the agent cluster check in [StructuredDeserialize](https://html.spec.whatwg.org/multipage/structured-data.html#structureddeserialize). We'd also need to add a check to the [`document.domain` setter](https://html.spec.whatwg.org/multipage/origin.html#dom-document-domain) to make it no-op. (Ideally we'd find some way of restructuring the existing checks in `document.domain` so that they are also tied to agent cluster boundaries.)
```

Comment:
The parenthetical was already there, but nothing in `document.domain` really cares or needs to care about those boundaries. Those checks are elsewhere, but they are also not affected as the origin is immutable and there can only be one agent cluster for it per browsing context group.

Reply:
Hmm, yeah, you're right. For the record, my intuition here was that it'd be kind of nice if instead of `document.domain` failing when:

- value is not a registrable domain suffix of and is not equal to the current effectiveDomain, or
- origin isolation is in place (and thus the check becomes value is not same-origin with the current origin)

the check was instead something like "value is outside the possible URLs that could end up in the current agent cluster".

But, there's no obvious way to derive that from existing spec constructs (you'd need to add a notion of "possible URLs that could end up in the current agent cluster"), and the different failure modes for the two cases means that you can't really unify the check anyway.

I'll remove.